### ISSUE TITLE
Renaming the latest version to avoid regressions

### ISSUE THREAD

**Issue by m4rc1e** [SorkinType/Merriweather#547]
At the end of last year, I finally finished mastering the latest sources, https://github.com/SorkinType/Merriweather/pull/542. Unfortunately, we felt that the changes were so large that is justified a new name (see https://github.com/google/fonts/pull/4077). @EbenSorkin I'm wondering if you have any thoughts on this?

We've previously had this issue with Crimson Text. We decided to release the update as Crimson Pro.

**Comment by m4rc1e** [comment]
ping @EbenSorkin 

cc @davelab6 @rosawagner

**Comment by EbenSorkin** [comment]
Parallel to this question, there are some fixes to Merriweather that really
should go in to the the old non-flex version. Things about Cyrillic letter
shapes especially but other things too. It might be worth separating things
that we would call 'major bug fixes' and things we would call
'changes/regressions'.

Marc - could we meet to discuss this next week sometime? Should we wait
until Dave is back? What makes best sense?

-e.

**Comment by davelab6** [comment]
Agreed, I believe subspacing the new Merriweather to its opsz-6, wdth-100, to get the full set of weights, should be a clean upgrade to the existing family. No need to wait for me, I think Marc can handle this among his other priorities in Q2.

**Comment by EbenSorkin** [comment]
There might be some nuance to that that Marc wants to assert. I'm open to
learning about it if that's the case.

-e.
